packages/aws-appsync-auth-link/src/signer/signer.ts

```diff
@@ -7,16 +7,23 @@ See the License for the specific language governing permissions and limitations
 */
 global.Buffer = global.Buffer || require('buffer').Buffer; // Required for aws sigv4 signing
 
-var url = require('url'),
-    crypto = require('aws-sdk/global').util.crypto;
+var url = require('url');
 
-var encrypt = function (key, src, encoding = '') {
-    return crypto.lib.createHmac('sha256', key).update(src, 'utf8').digest(encoding);
+var { Sha256 } = require('@aws-crypto/sha256-universal')
```

Comment:
Can we try to use https://github.com/aws-amplify/amplify-js/blob/main/packages/core/src/Signer.ts#L15 this approach instead, that hash is sync that will make this a non-breaking change. What do you think?